ui/app/components/oidc/key-form.js

```diff
@@ -22,6 +22,7 @@ import { task } from 'ember-concurrency';
  * @param {Object} model - oidc client model
  * @param {onCancel} onCancel - callback triggered when cancel button is clicked
  * @param {onSave} onSave - callback triggered on save success
+ * @param {boolean} [isModalForm=false] - boolean to determine if form should render a limited set of options. Used when rendered outside of OIDC provider workflow (ex: SearchSelectWithModal when mounting a WIF secret engine).
```

Comment:
Thank you for updating the jsdoc! I feel like we can be more specific here since it's only hiding one field. I also think it's helpful to explain what true and/or false does. 
```suggestion
 * @param {boolean} [isModalForm=false] - if true, hides inputs related to selecting an application which is only relevant to the OIDC provider workflow
```